components/eppp_link/README.md

```diff
@@ -0,0 +1,44 @@
+# ESP PPP Link component (eppp_link)
+
+The component provides a general purpose connectivity engine between two micro-controllers, one acting as PPP server (slave), the other one as PPP client (host). Typical application is a WiFi connectivity provider for chips that do not have WiFi:
+
+```
+             SLAVE micro                                  HOST micro
```

Comment:
I would still like to use the name `SLAVE` and `HOST`, as this is a well established terminology for `esp_hosted` like solutions, and this is an example of using `eppp_link` in this scenario.
So that, the `SLAVE micro` for example doesn't only implement `PPPoS server`, but also a WiFi client (+provisioning, maybe OTA) and NAT, etc.

Reply:
Actually, I'm also working on other examples that would use this PPP client/server, but add other functionality:
* `esp_modem` counter-part: app with `esp-at` and `eppp` -- a full featured WiFi modem device
* wifi connect -- socket based (g?)RPC to `esp_wifi` with a custom channel for Ethernet frames.

Reply:
So maybe adding a small diagram to the examples will help, I had to come back in code to understand better how the PPPoS would work on the examples. I suggest a second read on examples README to search for improvement (Sorry to not have concrete suggestions).

Reply:
Added some explanation and rephrased. PTAL again.